services/blockchain-indexer/config.js

```diff
@@ -213,4 +213,6 @@ config.estimateFees = {
 	bufferBytesLength: Number(process.env.ESTIMATES_BUFFER_BYTES_LENGTH) || 0,
 };
 
+config.invokeAllowedMethods = String(process.env.INVOKE_ALLOWED_METHODS).split(',') || [];
```

Comment:
If `INVOKE_ALLOWED_METHODS` is not explicitly declared, it'll result in `'undefined'`.
```suggestion
config.invokeAllowedMethods = String(process.env.INVOKE_ALLOWED_METHODS || '').split(',') || [
	'dynamicReward_getExpectedValidatorRewards',
	'token_hasUserAccount',
	'token_getInitializationFees',
	'interoperability_getMinimumMessageFee',
	'txpool_getTransactionsFromPool'
];
```

Reply:
Applied.